src/OrderETAs.ts

```diff
@@ -45,10 +50,30 @@ export class OrderETAs {
         }
       });
     });
-  }
-  private beautifyEta(etaSpan: Node){
-    const eta = convertDurationToETA(etaSpan.textContent);
+    }
+    */
+
+    private beautifyAggregateProductionQueue() {
+        const prodLines = Array.from(document.querySelectorAll(Selector.ProdLine));
```

Comment:
Select the table itself using the `Selector` constant map, then drill down using regular css selectors, and child properties

Reply:
just built my first duplicate building and it messes up the aggregation, will have to think of a sensible way to calculate. gonna refactor the selectors while at it.